rust/syntect-plugin/src/main.rs

```diff
@@ -731,7 +733,27 @@ impl<'a> Plugin for Syntect<'a> {
 }
 
 fn main() {
-    let syntax_set = SyntaxSet::load_defaults_newlines();
+    // get the Home dir (works with Windows / Linux / Mac)
+    let syntax_set = match dirs::home_dir() {
+        Some(path) => {
+            let mut path = String::from(path.to_str().unwrap());
+            // Set the path to the packfile.
+            path.push_str("/.config/xi/plugins/syntect/syntaxes.packfile");
```

Comment:
This shouldn't be necessary; that is, we don't want to be reading this from a file at runtime. Instead we want to use `include_bytes!` to include these syntaxes in the binary itself. To see how this works, you can go look at the `dumps.rs` file in syntect, which is where `load_defaults` is implemented.

Reply:
Check out my next commit.